src/machine/machined-varlink.c

```diff
@@ -726,7 +726,11 @@ static int manager_varlink_init_userdb(Manager *m) {
         if (m->varlink_userdb_server)
                 return 0;
 
-        r = varlink_server_new(&s, SD_VARLINK_SERVER_ACCOUNT_UID|SD_VARLINK_SERVER_INHERIT_USERDATA, m);
+        r = varlink_server_new(
+                        &s,
+                        SD_VARLINK_SERVER_ACCOUNT_UID|SD_VARLINK_SERVER_INHERIT_USERDATA|
+                        SD_VARLINK_SERVER_ALLOW_FD_PASSING_OUTPUT,
```

Comment:
hmm, do we send fds from the userdb server? i don't think we do?

Reply:
i dont think we do. plesae drop the flag from the userdb varlink server, it's sufficient if the other one has it.